modules/ROOT/partials/configuration/toolbar_sticky.adoc

```diff
@@ -17,6 +17,7 @@ To see a demo of sticky toolbar, see the https://codepen.io/tinymce/full/qBWLLMP
 
 include::partial$misc/differs_for_mobile.adoc[]
 
+[discrete]
```

Comment:
I don't think this should be used, no other page uses this discrete attribute. Other pages have the examples show up in the ToC.